packages/data-provider/src/config/config.ts

```diff
@@ -39,12 +43,14 @@ export type AppConfig = {
 	logger: LoggerConfig;
 	schemaService: SchemaServiceConfig;
 	idService: IdServiceConfig;
+	audit: AuditConfig;
```

Comment:
Since this AppConfig is part of the public interface for the Lyric Data-Pprovider, I am wondering if there is a better way to organize this feature. My two thoughts are:
1. Since we have default behaviour (audit is enabled) could we make this an optional part of the config, where people would need to intentionally disable it?
2. We can expect to have more optional features in the future, so instead of having a whole property just for `audit`, should we instead group these inside a property called `features`? Something like:

```ts
export type AppConfig = {
	db: DbConfig;		db: DbConfig;
	limits: LimitsConfig;		limits: LimitsConfig;
	logger: LoggerConfig;		logger: LoggerConfig;
	schemaService: SchemaServiceConfig;		schemaService: SchemaServiceConfig;
	idService: IdServiceConfig;		idService: IdServiceConfig;
	features?: { audit?: AuditConfig; }
}
```

Reply:
Also, please sort the properties in this type alphabetically.

Reply:
I've reorganized audit properties on to features, this will allow add additional features in the future